internal/server/handle_callback.go

```diff
@@ -0,0 +1,61 @@
+package server
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/jmartin82/mmock/pkg/mock"
+)
+
+// HandleCallback makes a callback required after a request
+func HandleCallback(cb mock.Callback) (*http.Response, error) {
```

Comment:
Nobody is using this error, what do you think about to remove the log error from here and add an error handler in the caller like:

```
	go func() {
		err := HandleCallback(mock.Callback)
		if err != nil {
			log.Printf("Error creating callback request: %s\n", err)
		}
	}()
```

Reply:
Yep that makes a lot of sense - thanks! I've kept the response as `(Response, error)` just because I think it does make the tests quite a bit better to be able to inspect the response, but can drop that if you'd prefer since it is only returned for tests. I've updated to no longer log in the function though - definitely a lot nicer.